code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -0,0 +1,256 @@
+/*
+
+CLOCKWORK CULT: Based off of the failed pull requests from vgstation
```

Comment:
I think there are three permanently closed PRs for this and one on hiatus.
I consider those three failed.